src/core/Table/Table.tsx

```diff
@@ -163,6 +164,7 @@ export const Table = <
     intersectionMargin = 300,
     onFilter,
     emptyFilteredTableContent,
+    filterTypes: userFilterFunctions,
```

Comment:
First I read this as `useFilterFunctions` which made me think it's a hook. 😄 

Maybe rename to something else?

Reply:
Renamed.